build/Nuget/Microsoft.NET.Nuget.proj

```diff
@@ -3,17 +3,13 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), 'Common.props'))\Common.props" />
 
   <PropertyGroup>
-    <NuGetOutDir>$(OutDir)Packages\</NuGetOutDir>
-    <OutDir>$(OutDir)Raw\</OutDir>
+    <InitialOutDir>$(OutDir)</InitialOutDir>
+    <NuGetOutDir>$(OutDir)Packages/</NuGetOutDir>
+    <OutDir>$(OutDir)Raw/</OutDir>
```

Comment:
Was it necessary to change the slashes here? It's inconsistent as outdir portion still has backslashes. If the final trailing slash is causing problems use trimend at invocation as we did for nuget pack.